glusterd2/volgen/templates/replicate.graph

```diff
@@ -0,0 +1,2 @@
+cluster/replicate
```

Comment:
This should be the same as the distreplicate graph, it needs to include cluster/distribute before cluster/replicate as well. All gluster volumes need to include distribute.

Reply:
Done